### PR TITLE
RN37015 - Token expose update

### PR DESCRIPTION
**Pull request by VanRobaysWouter**

### CODE REVIEW

File: user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md [hunk 1]
```diff
@@ -59,7 +59,13 @@ The entry point method has two parameters.
...
   | RawBody | `string` | The full body of the HTTP request as a string. This can be deserialized and used in the script. See [User input data](#user-input-data). |
   | Parameters | `Dictionary<string, string>` | Contains the deserialized parameters if you select *Dictionary (parsed from JSON)* when configuring the API. See [User input data](#user-input-data). |
-  | TokenId | `ApiTokenId` | Contains the ID of the [ApiToken](xref:UD_APIs_Objects_ApiToken) with which the API is triggered. Available from DataMiner 10.3.9/10.4.0 onwards<!-- RN 36856 -->. |
+  | Context | [ApiTriggerContext](#apitriggercontext) | Contains properties with info concerning the request. Available from DataMiner 10.3.9/10.4.0 onwards. <!-- RN 217125 --> |
+
+#### ApiTriggerContext
+
+  | Property | Type | Description |
+  |--|--|--|
+  | TokenId | `Guid` | Contains the ID of the [ApiToken](xref:UD_APIs_Objects_ApiToken) with which the API is triggered. Available from DataMiner 10.3.9/10.4.0 onwards<!-- RN 36856 --><!-- RN 217125 -->. |
```
Comment (Pieter-JanV) [inline user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md:68]:
```suggestion
  | TokenId | `Guid` | Contains the ID of the [ApiToken](xref:UD_APIs_Objects_ApiToken) with which the API is triggered. Available from DataMiner 10.3.9/10.4.0 onwards<!-- RN 37015 --><!-- DCP 217125 -->. |
```

File: user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md [hunk 2]
```diff
@@ -59,7 +59,13 @@ The entry point method has two parameters.
   | Route | `string` | The suffix of the URL that this API call is triggered with. Having this available makes it possible to reuse the same script for different routes. See [Route](#route).|
   | RawBody | `string` | The full body of the HTTP request as a string. This can be deserialized and used in the script. See [User input data](#user-input-data). |
   | Parameters | `Dictionary<string, string>` | Contains the deserialized parameters if you select *Dictionary (parsed from JSON)* when configuring the API. See [User input data](#user-input-data). |
-  | TokenId | `ApiTokenId` | Contains the ID of the [ApiToken](xref:UD_APIs_Objects_ApiToken) with which the API is triggered. Available from DataMiner 10.3.9/10.4.0 onwards<!-- RN 36856 -->. |
+  | Context | [ApiTriggerContext](#apitriggercontext) | Contains properties with info concerning the request. Available from DataMiner 10.3.9/10.4.0 onwards. <!-- RN 217125 --> |
```
Comment (Pieter-JanV) [inline user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md:62]: Doing the same change as the other one here?

**Review by Pieter-JanV — CHANGES_REQUESTED** [review]

File: user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md (hunk 1 above)
Comment (VanRobaysWouter) [reply user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md:68]:
```suggestion
  | TokenId | `Guid` | Contains the ID of the [ApiToken](xref:UD_APIs_Objects_ApiToken) with which the API is triggered. Available from DataMiner 10.3.9/10.4.0 onwards<!-- RN 36856 --><!-- RN 37015 -->. |
```

File: user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md (hunk 2 above)
Comment (VanRobaysWouter) [reply user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md:62]:
```suggestion
  | Context | [ApiTriggerContext](#apitriggercontext) | Contains properties with info concerning the request. Available from DataMiner 10.3.9/10.4.0 onwards. <!-- RN 37015 --> |
```

**Review by HansSolos — APPROVED** [review]

**Review by thomieboy — APPROVED** [review]

**Review by EdithLansens — APPROVED** [review]

**Pull request merged**